src/client/runner/automation/options.js

```diff
@@ -201,3 +201,23 @@ export class MoveOptions extends MouseOptions {
         ]);
     }
 }
+
+
+// Type
+export class TypeOptions extends ClickOptions {
+    constructor (obj, validate) {
+        super();
+
+        this.text    = '';
+        this.replace = false;
+
+        this._assignFrom(obj, validate);
+    }
+
+    _getAssignableProperties () {
+        return super._getAssignableProperties().concat([
+            { name: 'text' },
```

Comment:
Update. I believe text shouldn't be part of options, it should be an argument, like in action